src/marketplace/create_marketplace_boilerplate.ts

```diff
@@ -0,0 +1,186 @@
+
+import { input } from "@inquirer/prompts";
+import { Command } from "commander";
+
+import shell from "shelljs";
+
+const MARKETPLACE_BOILERPLATE_REPO_URL = "https://github.com/0xsequence/marketplace-boilerplate/";
+
+export async function createMarketplaceBoilerplate(program: Command, options: any) {
+    let walletType = options.walletType;
+    let waasConfigKey = options.waasConfigKey;
+    let projectAccessKey = options.projectAccessKey;
+    let googleClientId = options.googleClientId;
+    let projectId = options.projectId;
+    let userWantConfigureEnvs = "yes";
+    
+    if (!walletType) {
+        console.log("Please provide the Wallet Type for your project.");
+        console.log("You can use 'universal' or 'waas' as values.");
```

Comment:
Can you change this to provide some context:

> You can use 'waas' for Embedded Wallet and 'universal' for Universal Wallet.
> For more information on wallet types: https://docs.sequence.xyz/solutions/wallets/overview

Reply:
Done